src/components/DocumentEditor/DocumentPage.js

```diff
@@ -0,0 +1,80 @@
+import { Editor, DocumentPageList } from "./index.js";
+import { getSelectedDocument, updateDocument } from "../../service/api.js";
+import { getItem, setItem } from "../../utils/storage.js";
+import checkNewComponent from "../../utils/checkNewComponent.js";
+
+export default function DocumentPage({ $target, initialState, updateSideBar }) {
+    const self = this;
+    checkNewComponent(DocumentPage, self);
+
+    const $documentPage = document.createElement("div");
+    $documentPage.className = "documentPage";
+
+    this.state = initialState;
+
+    this.setState = async (nextState) => {
+        this.state = nextState;
+        const { documentId } = this.state;
+
+        postLocalSaveKey = `temp-post-${documentId}`;
+
+        if (documentId != null) {
+            const selectedDocument = await getSelectedDocument(
+                `/${documentId}`
+            );
+
+            editor.setState(selectedDocument);
+            documentPageList.setState(selectedDocument);
+        }
+
+        this.render();
+    };
+
+    let postLocalSaveKey = `temp-post-${this.state.documentId}`;
+
+    const post = getItem(postLocalSaveKey, {
+        title: "",
+        content: "",
+    });
+
+    const updateDocumentPage = async (post) => {
+        const { title } = post;
+        if (title == null || title.trim() === "") {
+            alert("제목을 작성해야 문서가 저장됩니다.");
+            return false;
+        }
+        await updateDocument(`/${post.id}`, post);
+        history.replaceState(null, null, `/documents/${post.id}`);
```

Comment:
```suggestion
        const { title, id } = post;
        if (title == null || title.trim() === "") {
            alert("제목을 작성해야 문서가 저장됩니다.");
            return false;
        }
        await updateDocument(`/${id}`, post);
        history.replaceState(null, null, `/documents/${id}`);
```

이왕 가져오는거 다 가져오면 요렇게 사용할 수 있을 것 같아요 ㅎㅎ

Reply:
아아 그렇네요 ㅠㅠㅠ 통일성 있게 작성해보겠습니다!!! 
세심하게 봐주셔서 감사합니다 😊🙌